zero_bin/leader/src/stdio.rs

```diff
@@ -1,42 +1,65 @@
-use std::io::{Read, Write};
+use std::io::Read;
+use std::sync::Arc;
 
-use anyhow::Result;
+use anyhow::{anyhow, Result};
 use paladin::runtime::Runtime;
 use proof_gen::proof_types::GeneratedBlockProof;
-use prover::{BlockProverInput, BlockProverInputFuture, ProverConfig};
+use prover::{BlockProverInput, ProverConfig};
+use tokio::sync::mpsc;
 use tracing::info;
 
+// Use some arbitrary number for the channel size, adjust if needed.
```

Comment:
This is a comment to devs right? We're not suggesting users change this are we?

Reply:
Also duplicate between here and in `leader/src.client.rs`, ideally should be unified.

Reply:
> This is a comment to devs right? We're not suggesting users change this are we?

Right